### DISCUSSION TITLE
CSS to run jsPsych on smartphones

### DISCUSSION THREAD

**Discussion by kurokida** [jspsych/jsPsych#3447] [Show and tell]
Hi all, 

I made a very simple CSS file to run jsPsych on smartphones.
I hope this is useful for someone.

https://github.com/kurokida/jspsych_smartphone_css

**Comment by jodeleeuw** [comment]
This looks fantastic. Thank you for sharing!

**Reply by kurokida** [reply]
Thank you for your reply. If you wish to include the CSS in the jsPsych repository, feel free to do so.
